ScoutSuite/providers/azure/rules/findings/sqldatabase-databases-no-auditing.json

```diff
@@ -1,11 +1,11 @@
 {
     "dashboard_name": "SQL Databases",
-    "description": "Auditing disabled",
-    "rationale": "You should enable auditing for all of your SQL databases. See CIS 4.2.1.",
+    "description": "Auditing is disabled for some SQL databases",
```

Comment:
`Auditing disabled for SQL databases` would be more in line with current descriptions